pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -67,6 +67,9 @@ buildPythonPackage rec {
     pytest-mock
     pytestCheckHook
     re-assert
+  ] ++ lib.optionals (!(stdenv.isDarwin && stdenv.isAarch64)) [
```

Comment:
Woul
```suggestion
  ] ++ lib.optionals (!trustme.meta.broken) [
```
work?

Reply:
No, this is enough use of `pyopenssl` that nix refuses to evaluate the package.